Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -1340,6 +1342,31 @@ sourceToDestinationConfig state = let
     }
   in sourceToDestinationConfig'
 
+chooseVehicleConfig :: ST.HomeScreenState -> ChooseVehicle.Config
+chooseVehicleConfig state = let
+  config = ChooseVehicle.config
+  chooseVehicleConfig' = config
+    { vehicleImage = "ny_ic_auto_quote_list"
+    , isSelected = true
+    , vehicleVariant = state.data.selectedEstimatesObject.vehicleVariant
```

Comment:
put `state.data.selectedEstimatesObject` inside a variable and use that in place of `state.data.selectedEstimatesObject`